packages/default-theme/components/SwCurrency.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="container" v-if="activeCurrency && availableCurrencies && availableCurrencies.length">
+    <SfSelect v-model="activeCurrency" :size="availableCurrencies.length" class="container__select">
```

Comment:
availableCurrencies should be loaded after a click on select, so first, loadAvailableCurrencies, before that we just displaying current currency from the context

Reply:
SfSelect needs to have all options when the activeCurrency is set. Because there is a reference to one from available currencies stick to the current one ;(